common/script/content/mystery-sets.js

```diff
@@ -129,6 +132,7 @@ let mysterySets = {
 
 each(mysterySets, (value, key) => {
   value.key = key;
+  value.text = t('mysterySet${key}'');
```

Comment:
These should be backticks (`), not single quotes ('). See http://babeljs.io/docs/learn-es2015/#template-strings

You also have an extra single quote in there.